spec/services/proofing/resolution/plugins/instant_verify_state_id_address_plugin_spec.rb

```diff
@@ -291,53 +291,23 @@
     subject(:proofer) { plugin.proofer }
 
     before do
-      allow(IdentityConfig.store).to receive(:proofer_mock_fallback).
-        and_return(proofer_mock_fallback)
       allow(IdentityConfig.store).to receive(:idv_resolution_default_vendor).
         and_return(idv_resolution_default_vendor)
     end
 
-    context 'when proofer_mock_fallback is set to true' do
-      let(:proofer_mock_fallback) { true }
+    context 'idv_resolution_default_vendor is set to :instant_verify' do
+      let(:idv_resolution_default_vendor) {          :instant_verify }
```

Comment:
I agree with Lauren that this is maybe... non-conventional? I'm not vehemently opposed to it if you think it's clearer though.

Reply:
I kinda feel vehemently opposed to it 😂

Reply:
I'm kinda surprised [Layout/SpaceInsideBlockBraces](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/SpaceInsideBlockBraces) isn't strict about how many spaces (just that it has at least one apparently?)

Reply:
what the heck